test/helpers/restricted-nori.ts

```diff
@@ -0,0 +1,95 @@
+import type { BigNumber } from 'ethers';
+
+import type { RestrictedNORI } from '@/typechain-types/contracts/RestrictedNORI';
+import { setupTest, expect } from '@/test/helpers';
+
+export const SECONDS_IN_1_YEAR_AVG = 31_556_952;
+export const SECONDS_IN_10_YEARS = 315_569_520;
+export const SECONDS_IN_5_YEARS = SECONDS_IN_10_YEARS / 2;
+
+export const setupTestLocal = global.hre.deployments.createFixture(
+  async (): Promise<Awaited<ReturnType<typeof setupTest>>> => {
```

Comment:
Is this not the same as 
```suggestion
  async (): ReturnType<typeof setupTest> => {
```